### ISSUE TITLE
Problem importing the Docker image?

### ISSUE THREAD

**Issue by verdurin** [pescobar/singularity-easybuild#6]
Just trying this with `Singularity 2.3.1` inside a `Vagrant` box and see an error when bootstrapping the image:
```
vagrant@vagrant:~/src$ sudo singularity create -s 4096 centos7-easybuild-3.4.1
Initializing Singularity image subsystem
Opening image file: centos7-easybuild-3.4.1
Creating 4096MiB image
Binding image to loop
Creating file system within image
Image is done: centos7-easybuild-3.4.1
vagrant@vagrant:~/src$ sudo singularity bootstrap centos7-easybuild-3.4.1 singularity-easybuild-pescobar/centos-7-easybuild-3.4.1.bootstrap
Sanitizing environment
Building from bootstrap definition recipe
Adding base Singularity environment to container
Docker image path: index.docker.io/library/centos:latest
ERROR Improperly formed manifest, layers or fsLayers must be present
```

**Comment by pescobar** [comment]
hi @verdurin 

few weeks ago docker did some change in the dockerhub api (I don't remember the details) and this broke the support in singularity 2.3 to pull images from dockerhub. See here https://github.com/singularityware/singularity/issues/925#issuecomment-329228235

this is fixed in latest singularity 2.4 but I still couldn't find the time to update this repo

**Comment by verdurin** [comment]
Yes, I've built my own Vagrant box with `Singularity 2.4` and it imports fine now, so it's definitely not a problem with your repo. Closing.
